minio/minio
# Allow metadata updates on meta bucket even in WORM mode

## Description
Allow metadata updates on meta bucket even in WORM mode

## Motivation and Context
This ensures that we can update the

- .minio.sys is updated for accounting/data usage purposes
- .minio.sys is updated to indicate if backend is encrypted
  or not.

## How to test this PR?
Start server with MINIO_WORM=on you can see that 
we do not update usage statistics anymore in worm mode.

## Types of changes

## Checklist:
